src/handlers/handlers.js

```diff
@@ -0,0 +1,17 @@
+var fs = require('fs');
+var path = require('path');
+
+var handlers = (module.exports = {});
+
+handlers.serveLanding = function(request, response) {
+  fs.readFile(
+    path.join(__dirname, '..', '..', 'public', 'index.html'),
+    function(err, file) {
+      if (err) {
+        throw err;
```

Comment:
Inconsistent use of ES5 features (`var`, `function(request, response) {})` which you don't use later - is this on purpose so that you convert to ES6 together in the code-along? I would prefer to stick to ES6 throughout  (which we didn't do for FAC10 as I retrospectively believe we should). The same applies to the router file below.